br/pkg/version/version.go

```diff
@@ -30,6 +30,9 @@ var (
 	compatibleTiFlashMajor4 = semver.New("4.0.0")
 
 	versionHash = regexp.MustCompile("-[0-9]+-g[0-9a-f]{7,}")
+
+	// check whether cross version
+	CrossVesion = false
```

Comment:
a global mutable variable? i don't think that's a good idea.

Reply:
> a global mutable variable? i don't think that's a good idea.

I have remove CrossVersion.